sys/aarch64/evec.S

```diff
@@ -145,7 +145,8 @@ ENTRY(handle_kern_trap)
         .cfi_signal_frame
         save_ctx 1
         mov     x0, sp
-        bl      kern_trap_handler
```

Comment:
Just as a notice - "magic constants", not sure if it would be better to define the values, from the enum below

Reply:
Yeah, I could move `kexc_reason_t` to a header and expose these constants through genassym, but I'm not sure if any of existing Aarch64 headers is suitable for this enum.